index.html

```diff
@@ -75,6 +75,16 @@ <h2>Autofill</h2>
 	<li><a href="./autofill/modal.html">Email autofill form within self-closing modal</a></li>
 	<li><a href="./autofill/form-submission.html">Form submission detection and autofill</a></li>
 	<li><a href="./autofill/frame-form-submission-parent.html">Form submission detection and autofill within an iframe</a></li>
+  <li>
+		Autoprompt
+		<ul>
+			<li><a href="./autofill/autoprompt/case-1.html">Case 1</a></li>
```

Comment:
It'd be good for those cases to explain them a bit, so that we don't have to rely on remembering the numbers e.g. "Form below the fold" instead of "Case 3".